scripts/scenarios/cloud7/qa/no-ssl/qa-scenario-2c.yaml

```diff
@@ -69,6 +69,7 @@ proposals:
 - barclamp: ceph
   attributes:
     disk_mode: first
+    client_network: admin
```

Comment:
huh? thats the opposite of what I'd expect. if mds are deployed, everything should be on the public network..